core/src/main/scala/org/apache/spark/rdd/CoalescedRDD.scala

```diff
@@ -58,7 +58,7 @@ private[spark] case class CoalescedRDDPartition(
       val parentPreferredLocations = rdd.context.getPreferredLocs(rdd, p.index).map(_.host)
       preferredLocation.exists(parentPreferredLocations.contains)
     }
-    if (parents.size == 0) 0.0 else (loc.toDouble / parents.size.toDouble)
```

Comment:
Thanks for your explanation. So there is a complex process to do the review. And it isn't worth it to do it with tiny PRs.